scripts/chair_info.py

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/python3
+
+""" Retrieves brother chair information
+
+boto library is required to be installed prior to usage and development.
+"""
+import boto3
+from boto3.dynamodb.conditions import Key, Attr
+import sys
+
+s3 = boto3.resource('s3')
+for bucket in s3.buckets.all():
+    print(bucket.name)
+
+TABLE_NAME = "Brothers"
+REGION_NAME = "us-east-2"
+
+# Creating the DynaoDB Client
+dynamodb_client = boto3.client('dynamodb', region_name=REGION_NAME)
+
+# Creating the DynamoDB Table Resource
+dynamodb = boto3.resource('dynamodb', region_name=REGION_NAME)
+table = dynamodb.Table(TABLE_NAME)
+
+resp = table.scan(
+    FilterExpression=Attr('Chair').contains(sys.argv[1])
```

Comment:
Since you are taking in input from command line, add an error handler to dismiss illegal amounts of arguments to prevent `IndexError`'s See the screenshot below
![Screen Shot 2020-11-02 at 1 03 16 PM](https://user-images.githubusercontent.com/16504480/97902548-c6acef00-1d0b-11eb-97e9-b28b34af7779.png)

In general it is a good habit to have all these error handlers in place. You can't assume the user will read your code nor use it correctly.